examples/application/abi/poll/contract.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "OpenPollingApp",
+    "methods": [
+        {
+            "name": "create",
+            "args": [
+                {
+                    "type": "string[]",
+                    "name": "options",
+                    "desc": "A list of options for the poll. This list should not contain duplicate entries."
+                },
+                {
+                    "type": "bool",
+                    "name": "can_resubmit",
+                    "desc": "Whether this poll allows accounts to change their submissions or not."
+                }
+            ],
+            "returns": {
+                "type": "void"
+            },
+            "desc": "Create a new polling application."
+        },
+        {
+            "name": "open",
+            "args": [],
+            "returns": {
+                "type": "void"
+            },
+            "desc": "Marks this poll as open.\nThis will fail if the poll is already open.\nThe poll must be open in order to receive user input."
+        },
+        {
+            "name": "close",
+            "args": [],
+            "returns": {
+                "type": "void"
+            },
+            "desc": "Marks this poll as closed.\nThis will fail if the poll is already closed."
+        },
+        {
+            "name": "submit",
+            "args": [
+                {
+                    "type": "uint16",
```

Comment:
nit: `uint8` (or even smaller) probably suffices. But there's no real requirement to limit user choices to 256.